linux/main.go

```diff
@@ -97,10 +97,15 @@ func main() {
 	w.SetTitle("Focalboard")
 	w.SetSize(1024, 768, webview.HintNone)
 
-	script := fmt.Sprintf("localStorage.setItem('focalboardSessionId', '%s');", sessionToken)
+	baseUrl := fmt.Sprintf("http://localhost:%d", port)
```

Comment:
`baseUrl` needs a trailing slash to make sure the port number can't have extra digits